cogs/dev.py

```diff
@@ -83,6 +89,37 @@ async def sync(self, ctx: commands.Context, guilds: commands.Greedy[discord.Obje
 
         await ctx.send(f"Synced the tree to {ret}/{len(guilds)}.")
 
+    @commands.group(aliases=['ext'])
+    @commands.has_permissions(administrator=True)
```

Comment:
Honestly, I thought this was already a cog check... Could you change this into a cog check? Also check against `manage_guild` instead of `administrator`, since I think that is the highest permission that the `SysAdmin` role has in the server.